pedsnetdcc/tests/db_test.py

```diff
@@ -132,6 +140,37 @@ def test_serial_execute_transaction(self):
         self.assertEqual(1, stmts[2].data[0][0])
 
 
+class StatementListTestVacuum(unittest.TestCase):
+
+    def setUp(self):
+        # Create a postgres database in a temp directory.
+        self.postgresql = Postgresql()
+        self.dburi = self.postgresql.url()
+        self.conn_str = make_conn_str(self.dburi)
+        # Reset the log handler.
+        handler.reset()
+
+    def tearDown(self):
+        # Destroy the postgres database.
+        self.postgresql.stop()
+
+    def test_serial_execute_vacuum_transaction(self):
+        stmts = StatementList()
+        stmts.append(Statement('VACUUM'))
+        stmts.serial_execute(self.conn_str, True)
+
+        self.assertIn('VACUUM cannot run inside a transaction block',
```

Comment:
I believe that `assertIn` was added in python 3.2, so this should be `self.assertTrue(foo in bar)` for python 2 compatibility.